### PR TITLE
chore(debug): add env/generate driver

### PR DESCRIPTION
**Pull request by kuojianlu**

### CODE REVIEW

File: packages/fx-core/resource/package.nls.json [hunk 1]
```diff
@@ -1042,6 +1042,7 @@
   "debug.warningMessage2": " You may be asked for your account credentials when installing the certificate.",
   "debug.install": "Install",
   "driver.spfx.error.noValidAppCatelog": "There is no valid app catalog in your tenant. You can update the property 'createAppCatalogIfNotExist' in app.yml to true if you want us to create it for you or you can create it by yourself.",
-  "driver.tools.error.invalidParameter": "Following parameter is missing or invalid for %s action: %s."
-
+  "driver.tools.error.invalidParameter": "Following parameter is missing or invalid for %s action: %s.",
+  "driver.env.error.invalidParameter": "Following parameter is missing or invalid for %s action: %s.",
```
Comment (swatDong) [inline packages/fx-core/resource/package.nls.json:1046]: Any idea to share one `invalidParameter` message? Or it's expected to create for each driver?

**Review by swatDong — APPROVED** [review]

File: packages/fx-core/resource/package.nls.json (hunk 1 above)
Comment (kuojianlu) [reply packages/fx-core/resource/package.nls.json:1046]: Currently each driver has its own error handling, so it is expected to create its own error type as well as the error message. However, we are happy if the driver framework provides some shared error hanlding logic afterwards.

**Review by XiaofuHuang — APPROVED** [review]

File: packages/fx-core/src/component/driver/env/generate.ts [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    try {
+      this.validateArgs(args);
+
+      if (args.target) {
+        await fs.ensureFile(args.target);
+        const envs = dotenv.parse(await fs.readFile(args.target));
+        const content = Object.entries({ ...envs, ...args.envs })
+          .map(([key, value]) => `${key}=${value}`)
+          .join(os.EOL);
+        await fs.writeFile(args.target, content);
```
Comment (XiaofuHuang) [inline packages/fx-core/src/component/driver/env/generate.ts:48]: If `.env` file contains comments, will they be still existing?

Comment (kuojianlu) [reply packages/fx-core/src/component/driver/env/generate.ts:48]: No. To improve it later if needed.

**Review by jayzhang — APPROVED** [review]

**Pull request merged**